### DISCUSSION TITLE
hi

### DISCUSSION THREAD

**Discussion by xing0831** [ROBlNET13/pvz#56] [General]
Can I use some of the pictures and code？In return, you can check it out [here](https://github.com/cutexingyun/cutexingyun.github.io)
Maybe there's something I can do to help you

**Comment by ROBlNET13** [comment]
Sure! Would it be alright if i took some of the plants like catapult ones or the corn cob or cat tail?

**Comment by ROBlNET13** [comment]
and maybe the miner zombie too? my game doesnt have them and i would like if i could have them

**Comment by xing0831** [comment]
Of course

**Comment by ROBlNET13** [comment]
Thank you!

**Comment by xing0831** [comment]
There may be bugs, please correct them

**Comment by ROBlNET13** [comment]
I managed to make the cat tail work for my game, its ok!
